pkg/ca/fileca/testdata/intermediate-cert.pem

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could we also add test data for a 3-long chain, with two intermediates and a root? This should exercise the code that validates a pool of roots and a pool of intermediates.